.github/workflows/build_scoped_rector.yml

```diff
@@ -0,0 +1,90 @@
+            -   run: composer remove phpstan/phpstan --no-update
+            -   run: composer config repositories.repo-name vcs https://github.com/phpstan/phpstan-src.git
+
+            -   run: composer require phpstan/phpstan-src:@dev --no-update
+            # mimics PHPStan https://github.com/phpstan/phpstan-src/blob/d3be969b13654394d0186b9597449d44477f0012/composer.json#L17
+            -   run: composer require jetbrains/phpstorm-stubs:dev-master#05d145c0bbafcf9a551fdd8824adb2a7e259fdaf --no-update
+            -   run: composer update --no-progress --ansi
+
+            # downgrade PHPStan code to PHP 7.2
+            -   run: bin/rector process vendor/phpstan/phpstan-src/src --config ci/downgrade-phpstan-php74-rector.php --ansi
+
+#            -   run: mkdir nested-rector
+            # @see https://unix.stackexchange.com/a/554912/74260
+            # copy to nested directory, so we have clear plate to work with
+            -   run: rsync -av * nested-rector
+
+            # see https://github.com/humbug/php-scoper#without-box
+            -   run: composer update --working-dir nested-rector --no-dev --prefer-dist --ansi
+
+            # 2. scope it
+            -   run: wget https://github.com/humbug/php-scoper/releases/download/0.13.9/php-scoper.phar
+            -
+                # to make paths in scoper.inc.php work, as the are relative
+                run: cd nested-rector && php ../php-scoper.phar add-prefix . --output-dir ../rector-scoped --config scoper.inc.php --force --ansi -vvv
+
+            -   run: bin/rector process rector-scoped/vendor/composer/platform_check.php --config ci/downgrade-php71-rector-scoped.php
+
+            -   run: composer dump-autoload --working-dir rector-scoped --ansi --optimize --classmap-authoritative
+
+            # 2.5 scope composer.json
+            -   run: vendor/bin/package-scoper scope-composer-json rector-scoped/composer.json --ansi
+
+            -   run: rm rector-scoped/phpstan.neon
+
+            # 3. run it
+            -   run: chmod 777 rector-scoped/bin/rector
+
+            # 4. get tag - see https://github.com/WyriHaximus/github-action-get-previous-tag
+            -
+                id: previous_tag
+                uses: "WyriHaximus/github-action-get-previous-tag@master"
+
+            # 5. publish it to remote repository
+            -
+                # Uses an action in the root directory
+                uses: symplify/github-action-monorepo-split@master
+                env:
+                    GITHUB_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+                with:
+                    package-directory: 'rector-scoped'
+                    split-repository-organization: 'rectorphp'
+                    split-repository-name: 'rector-prefixed'
+                    tag: ${{ steps.previous_tag.outputs.tag }}
+                    user-name: "kaizen-ci"
+                    user-email: "info@kaizen-ci.org"
```

Comment:
ohh I see why you asked this question about kaizen on twitter ;-)

Reply:
Also ;)... I was exploring if more developers know Japanese :)